### ISSUE TITLE
I cannot type a custom code for a plan

### ISSUE THREAD

**Issue by htmgarcia** [OSTraining/Simplerenew#416]
If I type a custom code, it returns an error:
<img width="573" alt="ostraining_-_administration_-_simple_renew__create_plan" src="https://cloud.githubusercontent.com/assets/4999794/13797346/70c45f76-ead2-11e5-8708-1ec15691ec67.png">
My guess tells me, SR auto generate the alias based on the plan's name and ignore the custom typed value for code.

**Comment by billtomczak** [comment]
What are you attempting to do? I see that that plan code is already in the system. It looks like you are trying to create a duplicate. Which is not allowed.

**Comment by htmgarcia** [comment]
Yes, but that's because I added the plan with the desired code in a *tricky way*.

To reproduce my initial report, try this, please:

- Create a new plan
- In plan's name type "Video (Yearly)" 
- In **Short code**: "something-unique"
- Save and close

You should get the error report saying that plan is duplicated.

**Comment by billtomczak** [comment]
okay - will check it out.

**Comment by billtomczak** [comment]
Unable to reproduce

**Comment by htmgarcia** [comment]
@billtomczak maybe this screen record may help: https://youtu.be/Kc6dXqfEggM

**Comment by billtomczak** [comment]
Got it now! The key was in the plan name. For a new plan it's keying on that instead of the plan code.
